src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -373,6 +374,18 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
     })
     await this.validate()
   }
+
+  /** Set validate on file and pay click. */
+  @Watch('getComponentValidate')
+  protected updateResolutionValidationDetail (): void {
+    // don't call validation during Jest tests because we are setting app valid
+    if (!this.isJestRunning) {
+      this.$refs.createSpecialResolutionRef.validate()
```

Comment:
I'm not quite sure why we want to use this ref and validate it here. There is "formValid" assigned to the v-model, and "lazy-validation" will validate the form. Let me know if I am missing something here?